scripts/hypothesistooling.py

```diff
@@ -194,7 +194,7 @@ def modified_files():
         for l in diff_output.split('\n'):
             filepath = l.strip()
             if filepath:
-                assert os.path.exists(filepath)
+                assert os.path.exists(filepath), filepath
```

Comment:
I put this in to debug some local problems I was having (for some reason, `check-release-file.py` had vanished and I was getting the Python header prepended to our requirements file), but I don't think it's a bad change, so I'm going to leave it in.